11ty/definitions/derpy.md

```diff
@@ -0,0 +1,29 @@
+---
+title: derpy
+slug: derpy
+flag:
+  text: 'Ableist Term'
+  level: avoid
+defined: true
+speech: adjective
+alt_words:
+  - ridiculous
+  - clumsy
+  - foolish
+  - misshapen
+  - malformed
+---
+
+clumsy, foolish, unattractive, or otherwise less intelligent person; 
```

Comment:
```suggestion
clumsy, foolish, unattractive, or otherwise unintelligent act or person.
```

Reply:
Add something like "used to mimic sounds made by people with intellectual/development disabilities"